src/plugins/condition/utils/styleUtils.js

```diff
@@ -62,17 +62,72 @@ const styleProps = {
     }
 };
 
+const aggregateStyleValues = (accumulator, currentStyle) => {
+    const styleKeys = Object.keys(currentStyle);
+    const properties = Object.keys(styleProps);
+    properties.forEach((property) => {
+        if (!accumulator[property]) {
+            accumulator[property] = [];
+        }
+        const found = styleKeys.find(key => key === property);
+        if (found) {
+            accumulator[property].push(currentStyle[found]);
+        }
+    });
+    return accumulator;
+};
+
+// Returns a union of styles used by multiple items.
+// Styles that are common to all items but don't have the same value are added to the nonSpecific list
+export const getConsolidatedStyleValues = (multipleItemStyles) => {
+    let aggregatedStyleValues = multipleItemStyles.reduce(aggregateStyleValues, {});
+
+    let styleValues = {};
+    let nonSpecific = [];
```

Comment:
`nonSpecific` are the styles that are different between the selected items? Maybe we should use the vernacular used elsewhere and name this `mixedStyles` or something?

Reply:
Agree on `mixedStyles` vs. `nonSpecific`

Reply:
I used nonSpecific as that's the prop that the toolbar controls were using before.
I'll change the new variables to mixedStyles and leave the old props untouched.

Reply:
Done.